### PR TITLE
fix invalid coins issue

### PR DESCRIPTION
**Pull request by wc117**

### CODE REVIEW

File: integration_tests/cw20.py [hunk 1]
```diff
@@ -13,10 +13,10 @@
 
 async def main():
     paloma = AsyncLCDClient(url="https://lcd.testnet.palomaswap.com/", chain_id="paloma-testnet-13")
-    paloma.gas_prices = "1ugrain"
+    paloma.gas_prices = "0.01ugrain"
```
Comment (verabehr) [inline integration_tests/cw20.py:16]: is this correct? I thought there are no decimals for grain/ugrain

Comment (wc117) [reply integration_tests/cw20.py:16]: I thought so. But the gas price works. As the result gas_limit * 0.01 =gas_limit / 100 = real gas spend.
https://paloma.explorers.guru/transaction/AF774F158BEDEDE07DCCD482A95170E159FEE1F7C300C514D52838641C3872B7

Comment (wc117) [reply integration_tests/cw20.py:16]: And why our Paloma's decimals are 6 digits only?

Comment (wc117) [reply integration_tests/cw20.py:16]: And I think 1 grain = 10 ** 6 ugrain. And grain has 6 decimals. ugrain doesn't have decimals.

**Pull request merged**